helm-charts/example-apiserver/templates/_helpers.tpl

```diff
@@ -67,4 +67,14 @@ Templates konk name
 */}}
 {{- define "example-apiserver.konkname" -}}
 {{- .Values.konk.name | default (printf "%s-konk" .Release.Name) }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Templates konk-service name
+*/}}
+{{- define "example-apiserver.konk-service-name" -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}-api
+{{- end }}
+{{- define "example-apiserver.konk-service-fullname" -}}
+{{- include "example-apiserver.konk-service-name" . }}-konk-service
```

Comment:
`konk-service` 🤮

Reply:
contacting all our users for a secret name change 🤮